src/olympia/blocklist/migrations/0035_alter_blockversion_soft.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.16 on 2024-10-07 12:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blocklist', '0034_remove_blockversion_hard_blockversion_soft'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blockversion',
+            name='soft',
+            field=models.BooleanField(choices=[(False, 'blocked'), (True, 'soft_blocked')], default=False),
```

Comment:
Regenerate the migration, it should be `field=models.BooleanField(choices=[(0, 'Blocked'), (1, 'Soft-Blocked')], default=False),`

Reply:
Done.